ergotree-interpreter/src/eval/bin_op.rs

```diff
@@ -591,10 +609,10 @@ mod tests {
         }
 
         #[test]
-        fn test_and_or(l in any::<bool>(), r in any::<bool>()) {
-            prop_assert_eq!(eval_relation_op(RelationOp::And, l, r), l && r);
-            prop_assert_eq!(eval_relation_op(RelationOp::Or, l, r), l || r);
+        fn test_and_or_xor(l in any::<bool>(), r in any::<bool>()) {
+            prop_assert_eq!(eval_logical_op(LogicalOp::And, l, r), l && r);
+            prop_assert_eq!(eval_logical_op(LogicalOp::Or, l, r), l || r);
+            prop_assert_eq!(eval_logical_op(LogicalOp::Xor, l, r), (l || r) && !(l && r));
```

Comment:
```suggestion
            prop_assert_eq!(eval_logical_op(LogicalOp::Xor, l, r), l ^ r);
```
Rust has logical XOR operator - https://doc.rust-lang.org/reference/types/boolean.html#logical-xor. Please, use it in the eval itself as well.

Reply:
Of course, I should have tested that!  Fixed.